vars/Debian.yml

```diff
@@ -1,3 +1,6 @@
 ---
 # Debian / Ubuntu specific variables
 postfix_ca_bundle: /etc/ssl/certs/ca-certificates.crt
+
+# Uncomment this if postfix is having issues in resolving DNS.
+enable_resolv_conf_watcher: false
```

Comment:
Could we define this in the role's defaults? Then the when clauses can simply be `when: enable_resolv_conf_watcher`. AFAIK we're not actually doing anything ubuntu specific here; i'm pretty sure the systemd configs would be identical for redhat too.

Reply:
@dale-c-anderson : Done

Reply:
:+1: 

Can I ask for a couple more items: 
- [ ] Add `state: "{{ enable_resolv_conf_watcher | ternary("present", "absent") }}"` to the template tasks, 
- [ ] Do a similar thing to the `enabled` attrirbute of the systemd tasks,
- [ ] In the molecule test variables, set the variable to true so we know if the thing will blow up or not

Reply:
I get `fatal: [instance]: FAILED! => {"changed": false, "msg": "'state' cannot be specified on a template"}` while adding it to the template tasks

Reply:
OK skip that part then; i've obviously mixed up my ansible modules. Besides, I was think the enabled: false of the systemd tasks might fail if the templates aren't there anyway.

Reply:
How do I pass the 'Idempotence' test?

Reply:
It's the [dameon_reload](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html#parameter-daemon_reload) attribute - it always reloads. So the attribute makes the systemd task not idempotent  :angry:  

To get around it you'll need to register each of the the template tasks, eg
```
template: 
  ...
when: ...
register: foo_result_1
```
and then add to the `when:` clause on the systemd tasks,
```
systemd: ... 
  ...
when: enable_resolv_conf_watcher
  and foo_result_1.changed
```